game-core/src/system/ally.rs

```diff
@@ -190,13 +194,21 @@ impl<'s> System<'s> for Spawner {
                     current_frame: 0,
                 };
 
+                let max_hp = config.ally.max_hp;
```

Comment:
It might make it easier to add new fields to this in the future if you use the following syntax

```rust
let config::Ally { max_hp, .. } = config.ally
```

Reply:
hmm, I can see that being more helpful in more complicated contexts. I don't know that helps that much here and now